rust-runtime/aws-smithy-http-server/examples/README.md

```diff
@@ -0,0 +1,25 @@
+# Smithy Rust Server SDK example
```

Comment:
This file's lines are not hard-wrapped.

Reply:
I don't understand what do you mean with this.

Reply:
I mean usually we treat Markdown files like code and have line lengths be set at max 80, 100, 120 characters etc.

The process of inserting newlines to keep line lengths under a threshold is called (hard-)wrapping text. Soft-wrapping is performed by the viewing application e.g. a Vim buffer soft-wraps text so that you don't have to scroll horizontally to see the ends of long lines.

Reply:
Got it. It was unclear because the comment is on the first like, which is less than 80 characters :D